x-pack/legacy/plugins/epm/common/constants.ts

```diff
@@ -17,4 +17,5 @@ export const PLUGIN = {
   CONFIG_PREFIX: 'xpack.epm',
 };
 
-export const SAVED_OBJECT_TYPE = 'epm';
+export const SAVED_OBJECT_TYPE_PACKAGES = 'epm-packages';
```

Comment:
Nit: I would call it `epm-package` like `index-pattern`. No need to change it in this PR.

Reply:
Good point. I'll change it now, that's one breaking change less.